contracts/src/models/npc.cairo

```diff
@@ -47,63 +47,36 @@ fn random_role(source: u128) -> u8 {
 #[derive(Serde, Copy, Drop, Print)]
 struct Characteristics {
     age: u8,
-    hunger: u8,
-    happiness: u8,
-    belligerency: u8,
     role: u8,
     sex: u8,
 }
 
 fn random_characteristics(randomness: @Span<u128>) -> Characteristics {
```

Comment:
Do we still need this function if we generate sex and role within the GPT prompt ?